ansible/roles/debops.nginx/templates/etc/nginx/nginx.conf.j2

```diff
@@ -51,9 +54,6 @@ http {
 
         add_header X-Clacks-Overhead "GNU Terry Pratchett";
 
-{% if nginx_http_options|d() and nginx_http_options %}
-{{ nginx_http_options | indent(8, true) | regex_replace("(?m)^\s*$", "") }}
-
 {% endif %}
```

Comment:
You should move the corresponding `endif` as well, otherwise the condition scope is incorrect.